src/guide/component-basics.md

```diff
@@ -319,11 +321,7 @@ Now `v-model` should work perfectly with this component:
 <custom-input v-model="searchText"></custom-input>
 ```
 
-Another way of creating the `v-model` capability within a custom component is to use the ability of `computed` properties' to define a getter and setter.
-
-In the following example, we refactor the `custom-input` component using a computed property.
-
-Keep in mind, the `get` method should return the `modelValue` property, or whichever property is being using for binding, and the `set` method should fire off the corresponding `$emit` for that property.
+Another way of implementing `v-model` within this component is to use the ability of `computed` properties to define a getter and setter. The `get` method should return the `modelValue` property and the `set` method should emit the corresponding event:
```

Comment:
I've condensed 3 paragraphs down to 1 but I don't think anything important has been lost.

Previously the wording made reference to the possibility of binding other properties with `v-model`. That concept hasn't been introduced yet and I don't think there's much value in mentioning it here.